bus/manager.go

```diff
@@ -0,0 +1,122 @@
+package bus
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/kairos-io/kairos-sdk/types"
+	"github.com/mudler/go-pluggable"
+)
+
+func NewBus(withEvents ...pluggable.EventType) *Bus {
```

Comment:
This basically provides a generic BUS with bells and whistles so we can reuse it from other places instead of each project needing to create its own manager. Provides options for logs, prefixes, events and so on so it should cover most of the usecases.